dockerfiles/sandbox/start-with-nginx.sh

```
@@ -1,25 +1,354 @@
+socket-timeout = 300
+disable-logging = false
+log-date = true
+log-prefix = [worker${i}]
+logto = /var/log/worker${i}.log
+EOF
+
+    if [ -n "$UWSGI_CHEAPER" ]; then
+        echo "cheaper = ${UWSGI_CHEAPER}" >> /tmp/worker${i}_uwsgi.ini
+    fi
+
+    > /var/log/worker${i}.log
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)